src/test/components/__snapshots__/SampleTooltipContents.test.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`SampleTooltipContents renders the sample tooltip properly 1`] = `
 <div
   class="tooltip"
   data-testid="tooltip"
-  style="--tooltip-detail-max-width: 600px; left: 21px; top: 10px;"
+  style="--tooltip-detail-max-width: 600px; left: 11px; top: 10px;"
```

Comment:
I'm not 100% sure of the reason for this. I _think_ this is because now we use the same boxes for `getBoundingClientRect` and `ReactDOM.findDOMNode`, and previously this wasn't possibly the case. It's difficult to return different values for eg the container and the screenshot node itself...

Reply:
Yeah, that looks odd. I also see the same changes in other tests. But manual tests doesn't show anything for me. So it looks good.